README.md

```diff
@@ -65,6 +65,10 @@ ops/
 Run the script `setup` in the directory which should be prepared and the
 necessary files are copied from the directory `skel/`.
 
+Then you must change the files according to your needs. In some files "HINT"'s
```

Comment:
I think you mean the plural form, so it should be `"HINT"s` instead of `"HINT"'s`.

Reply:
fixed